lib/opbeat/rack.rb

```diff
@@ -28,16 +28,14 @@ def call(env)
       rescue Error => e
         raise # Don't capture Opbeat errors
       rescue Exception => e
-        evt = Event.from_rack_exception(e, env)
-        Opbeat.send(evt)
+        Opbeat.capture_rack_exception(e, env)
```

Comment:
Also, maybe I didn't explain properly. This PR allows users to call the `capture_rack_exception` from their code (similar to `capture_exception`), allowing them to pass rack environment.